src/main/resources/templates/Policies/instantEmailBody.html

```diff
@@ -0,0 +1,114 @@
+{@java.time.LocalDateTime timestamp}
+{@com.redhat.cloud.notifications.ingress.PoliciesParams params}
+{@java.util.List<com.redhat.cloud.notifications.ingress.Tag> tags}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Policy Notification</title>
+</head>
+<body>
+<div width="100%" style="margin:0;padding:0!important;font-family:overpass,arial;font-size:20px;box-sizing:border-box">
+    <table border="0" cellpadding="0" cellspacing="0" width="100%">
+        <tbody>
+        <tr>
+            <td>
+                <table align="center" border="0" cellpadding="0" cellspacing="0" width="600"
+                       style="border-collapse:collapse">
+                    <tbody>
+                    <tr>
+                        <td align="center" bgcolor="#000000" style="padding:20px 0 20px 0">
+                            <a href="https://cloud.redhat.com/insights/" style="display:block">
+                                <img src="https://cloud.redhat.com/apps/insights/images/email/rh-insights-logo.png"
+                                     alt="Red Hat Insights logo" width="152" height="60" style="display:block">
+                            </a>
+                        </td>
+                    </tr>
+
+                    <tr>
+                        <td bgcolor="#2c2c2" width="600" valign="top">
+                            <div width="100%" align="center"
+                                 style="padding:20px 20px 20px 20px;color:#ffffff;font-size:18px;font-weight:bold">
+                                Policies triggered - {timestamp}
+                            </div>
+                        </td>
+                    </tr>
+
+                    <tr>
+                        <td bgcolor="#f5f5f5" style="padding:20px 20px 0 20px">
+                            <table border="0" cellpadding="0" cellspacing="0" width="100%">
+                                <tbody>
+                                <tr>
+                                    <td bgcolor="ffffff" style="padding:20px 20px 20px 20px;font-size:16px">
+                                        Policy trigger notification for <strong>{tags.getFirst("display_name")}</strong>.<br>
+                                        System checked in at {timestamp} and triggered {params.triggers.size()} {#if params.triggers.size() is 1}policy{#else}policies{/if}.
```

Comment:
This timestamp is wrong and needs to come out of the engine when it processed the message. Or even better from the original payload.

Reply:
IIRC it comes from the sender of the notification. I'll verify it.

Reply:
Yes, that comes from the sender: https://github.com/RedHatInsights/policies-engine/blob/master/external/src/main/java/com/redhat/cloud/policies/engine/actions/plugins/WebhookActionPluginListener.java#L63
So is the time that the sender uses.

Reply:
Ok. I think the real issue is in the engine. I recall an open ticket